src/translations/en.json

```diff
@@ -1635,7 +1635,10 @@
           "zigbee_information": "View the Zigbee information for the device."
         },
         "confirmations": {
-          "remove": "Are you sure that you want to remove the device?"
```

Comment:
By changing this from a string to an object translations will override the new object with the string as long as they are not updated causing us to no longer have a fallback to english. (as `remove.title` would not longer exist, as `remove` would be a string)

Reply:
Si may be I should use remove_title and remove_test keys? 😕

Reply:
That would work yeah...